### PR TITLE
Add new make commands

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,13 +2,21 @@ start-deps:
 	docker-compose up -d
 	direnv reload
 
+start: start-deps
+	yarn build && node lib/entrypoint/graphql.js | yarn pino-pretty -c -l
```
Comment (bodymindarts) [inline Makefile:6]: Don't you need to source `.envrc` here too?

File: Makefile [hunk 2]
```diff
@@ -2,13 +2,21 @@ start-deps:
...
+
+watch:
+	yarn nodemon -V -e ts,graphql -w ./src -x make start
+
 clean-deps:
 	docker-compose rm -sfv
 
 reset-deps: clean-deps start-deps
 
+restart-integration: reset-deps integration
```
Comment (bodymindarts) [inline Makefile:16]: Is there a need for both `restart-integration` and `integration`? Serious question in your workflow do you ever need both? If not I think we can just use `integration`:
```
integration: restart-deps
  . ./.envrc && yarn jest --forceExit --bail --runInBand --verbose $$TEST | yarn pino-pretty -c -l
```

File: Makefile (hunk 1 above)
Comment (krtk6160) [reply Makefile:6]: It works without it. Could you try it out on your machine too?

File: Makefile (hunk 2 above)
Comment (krtk6160) [reply Makefile]: Done

**Pull request merged**